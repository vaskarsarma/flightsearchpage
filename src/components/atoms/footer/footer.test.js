import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import Footer from "./footer";

describe("<Footer />", () => {
  let container;

  beforeEach(() => (container = shallow(<Footer />)));

  it("<Footer /> Rendered Correctly", () => {
    expect(shallowToJson(container)).toMatchSnapshot();
  });

  it("should render a <footer />", () => {
    expect(container.find("footer").length).toEqual(1);
  });
});
