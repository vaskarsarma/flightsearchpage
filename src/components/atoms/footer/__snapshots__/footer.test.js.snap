// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> <Footer /> Rendered Correctly 1`] = `
<footer
  className="App-footer"
/>
`;
